Backend/app/rider-platform/rider-app/Main/src/Domain/Constants/ChatSuggestions.hs

```diff
@@ -0,0 +1,165 @@
+module Domain.Constants.ChatSuggestions where
+
+import qualified Data.Map.Strict as Map
+import Kernel.Prelude
+
+data Messages = Messages
+  { en_us :: Text,
+    ta_in :: Text,
+    kn_in :: Text,
+    hi_in :: Text,
+    ml_in :: Text,
+    bn_in :: Text
+  }
+  deriving (Show)
+
+dataMap :: Map.Map Text Messages
+dataMap =
+  Map.fromList
```

Comment:
Just save datamap here not the whole file, Messages type you can keep in domain type